explainer.py

```diff
@@ -0,0 +1,40 @@
+import os
+import asyncio
+import async_client
+import json_file
+import presentation_parser
+
+UPLOAD_FOLDER = 'uploads'
+OUTPUT_FOLDER = 'outputs'
+
+
+async def process_files():
+    """
+    Searches for new pptx files in the uploads folder
+     and sends them to async_client and then saves the json in the outputs folder
+    """
+    while True:
+        for filename in os.listdir(UPLOAD_FOLDER):
+            if filename.endswith('.pptx'):
+                file_path = os.path.join(UPLOAD_FOLDER, filename)
+                list_presentation = presentation_parser.extract_text(file_path)
+                if list_presentation == [""]:
+                    # Error loading the presentation
+                    os.remove(file_path)
```

Comment:
If you get to a situation that you split you functions to parts using newlines, just split it to functions instead :)